modules/serving-transport-encryption-configuring-selfsigned-clusterissuer.adoc

```diff
@@ -0,0 +1,70 @@
+:_content-type: PROCEDURE
+[id="serving-transport-encryption-configuring-selfsigned-clusterissuer_{context}"]
+= Configuring a SelfSigned ClusterIssuer
+
+[IMPORTANT]
+====
+For simplicity, this procedure uses a `SelfSigned` issuer as root certificate. However, be aware of the implications and limitations of this method documented in the "SelfSigned" section of the `cert-manager` documentation.
+
+If you run your company-specific Private Key Infrastructure (PKI), Red Hat recommends the CA issuer. Refer to the cert-manager documentation on CA issuers for more details. However, you can use any other issuer that allows signing of certificates as described in the "Choice of a certificate issuer" section.
+====
+
+.Procedure
+
+. Create a `SelfSigned` `ClusterIssuer`:
++
+[source,yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: knative-serving-selfsigned-issuer
+spec:
+  selfSigned: {}
+----
```

Comment:
```suggestion
. Create a `SelfSigned` `ClusterIssuer` custom resource (CR):
+
.Example ClusterIssuer CR
[source,yaml]
----
apiVersion: cert-manager.io/v1
kind: ClusterIssuer
metadata:
  name: knative-serving-selfsigned-issuer
spec:
  selfSigned: {}
----
```
- Fix formatting per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references
- Add code block titles, see examples https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#source-tags-for-terminal-commands-and-output